pkg/systemstatsmonitor/disk_collector.go

```diff
@@ -300,6 +316,34 @@ func listRootBlockDevices(timeout time.Duration) []string {
 	return strings.Split(strings.TrimSpace(string(stdout)), "\n")
 }
 
+// recordBootDiskSize records the disk block devices that are not a slave or holder.
```

Comment:
Insensitive language (unfortunately also in lsblk docs). Suggest leaving it out (here and line 324) or using an alternative (secondary, follower etc)

Reply:
updated the comment